examples/webgl_loader_sea3d_hierarchy.html

```diff
@@ -120,6 +116,9 @@
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				renderer.setClearColor( 0x333333, 1 );
+				renderer.shadowMap.enabled = true;
+				renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+				renderer.shadowMap.cullFace = THREE.CullFaceBack;
```

Comment:
Yes, certainly is.

You can also call `setDuration` passing how much global mixer time your animation is supposed to take as a more high-level way of setting `.timeScale`.

The value of `.timeScale` is effective when the action is not paused and not being time warped. The effective time scale during warping can be queried with `getEffectiveTimeScale`. The result is zero when the action is paused. The corresponding setter stops all warping, but does not release the `.paused` state.